console/console-init/ui/src/components/Editor/Editor.tsx

```diff
@@ -9,20 +9,34 @@ const Editor: React.FunctionComponent<IAceEditorProps> = ({
   name,
   onChange,
   enableBasicAutocompletion,
-  enableLiveAutocompletion
+  enableLiveAutocompletion,
+  width = "auto",
+  className,
+  height,
+  theme = "github",
+  fontSize = 14,
+  placeholder,
+  defaultValue,
+  maxLines = 100
 }) => {
   return (
     <AceEditor
       mode={mode}
-      theme="github"
-      fontSize={14}
+      theme={theme}
+      fontSize={fontSize}
```

Comment:
we can use fontSize as fontSize={fontSize||14} and pass the font size as props. Same with theme.

Reply:
This will work in same way as expected. Only way of writing is different.
Either you can define default value on top or where you use.